tool/src/main/java/org/wildfly/security/tool/Command.java

```diff
@@ -204,6 +221,28 @@ public void printDuplicatesWarning(CommandLine cmdLine, List<String> duplicatesA
         }
     }
 
+    /**
+     * Prints out a warning message if silentMode is not enabled and adds the warning to the summary
+     * if summaryMode is enabled
+     *
+     * @param warning The warning to be shown
+     */
+    protected void warningHandler(String warning) {
+        System.out.print("WARNING: ");
+        System.out.println(warning);
+    }
+
+    /**
+     * Determines if a summary needs to be printed and prints summary after an error is thrown
```

Comment:
Similar comment here, this method doesn't determine if the summary needs to be printed, right. It looks like it just sets the status.